src/main/kotlin/lotto/view/InputView.kt

```diff
@@ -0,0 +1,26 @@
+package lotto.view
+
+import lotto.constants.Messages
+
+/**
+ * 입력을 받는 클래스, 기본적인 필터링만 한다.
+ * Created by Jaesungchi on 2022.05.24..
+ */
+object InputView {
+    fun getPrice(readStringValue: () -> String? = { readlnOrNull() }): Int {
+        println(Messages.WRITE_YOUR_MONEY)
+        return changeStringToInt(readStringValue())
+    }
+
+    fun getWinningNumber(readStringValue: () -> String? = { readlnOrNull() }): String {
+        println(Messages.WRITE_WINNING_NUMBER)
+        val input = readStringValue()
+        require(!input.isNullOrBlank()) { Messages.Error.INPUT_IS_NULL_OR_BLANK }
+        return input
+    }
+
+    private fun changeStringToInt(input: String?): Int {
+        require(!input.isNullOrBlank()) { Messages.Error.INPUT_IS_NULL_OR_BLANK }
+        return input.toIntOrNull() ?: throw IllegalArgumentException(Messages.Error.INPUT_IS_NOT_NUMBER)
```

Comment:
```suggestion
        return input.toIntOrNull() ?: throw IllegalArgumentException(Messages.Error.INPUT_IS_NOT_NUMBER)
```

`Int` 외의 모든 것을 `null`을 만들어주면, `INPUT_IS_NOT_NUMBER` 로도 충분할 것 같아요